diesel/src/pg/expression/expression_methods.rs

```diff
@@ -2,6 +2,9 @@ use expression::{AsExpression, Expression};
 use super::operators::*;
 use types::{Array, Text};
 
+/// PostgreSQL expression methods which are not type specific.
+///
+/// This trait is implemented for all types that implement `Expression`.
```

Comment:
Hmm, a general note: I think docs like

> This trait is implemented for […]

are pretty much worthless for people who know how to use rustdoc. They only appear on detail pages for the trait, and these pages include a list of implementors (see [rendered docs](https://docs.rs/diesel/0.16.0/diesel/expression_methods/trait.PgExpressionMethods.html#implementors)).

_But_ they may be useful in other places, e.g., tooltips in IDEs; or, when they convey more the indent of which types are expected to implemented this (like in `SortExpressionMethods`).